java-checks/src/main/java/org/sonar/java/checks/IndentationCheck.java

```diff
@@ -131,14 +131,17 @@ public void visitMethodInvocation(MethodInvocationTree tree) {
 
   @Override
   public void visitLambdaExpression(LambdaExpressionTree lambdaExpressionTree) {
-    int previousTokenLine = getPreviousToken(lambdaExpressionTree).line();
-    int lambdaFirstTokenLine = lambdaExpressionTree.firstToken().line();
-    if (previousTokenLine != lambdaFirstTokenLine) {
-      expectedLevel += indentationLevel;
-    }
-    super.visitLambdaExpression(lambdaExpressionTree);
-    if (previousTokenLine != lambdaFirstTokenLine) {
-      expectedLevel -= indentationLevel;
+    // scan lambdaExpressionTree.parameters() is useless
```

Comment:
Yes, that was actually the point : the comment is useless if you don't explain the why.